webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -244,8 +244,8 @@ def bugzilla_submit_url(report, bug_product):
         'product': bug_product,
         'op_sys': op_sys,
         'rep_platform': report['cpu_name'],
-        'cf_crash_signature': '[@ {}]'.format(report['signature']),
-        'short_desc': 'Crash in {}'.format(report['signature']),
+        'cf_crash_signature': u'[@ {}]'.format(report['signature']),
+        'short_desc': u'Crash in {}'.format(report['signature']),
```

Comment:
I'm not sure what you mean by that. I think the resulting value should be a string--not a unicode. So you can solve this problem two ways:

```
'{}'.format(u'unicode sträng')
```

You can either turn the `'{}'` into a unicode so there's no encoding going on or you can convert the `u'unicode sträng'` into a string using something like `smart_str`.

I think we should be doing the second one since then we don't have to deal with changing how we're doing things with Bugzilla which isn't tested and who knows if it could change.

So I think the better change to make is something like this:

```
+        'cf_crash_signature': '[@ {}]'.format(smart_str(report['signature'])),
+        'short_desc': 'Crash in {}'.format(smart_str(report['signature'])),
```

Does that make sense? If you want, I can write a `smart_str` for Socorro and put it in `socorro/lib/`.